src/static/riot/competitions/detail/_tabs.tag

```diff
@@ -314,7 +314,9 @@
                     })
                 })
             })
-            self.selected_leaderboard_index = self.competition.leaderboards[0].id
+            if (self.competition.leaderboards.length > 0) {
+                self.selected_leaderboard_index = self.competition.leaderboards[0].id
```

Comment:
could do this a couple different ways:
```javascript
self.selected_leaderboard_index = _.get(_.head(self.competition.leaderboards), 'id')
```
or
```javascript
if (!_.isEmpty(self.competition.leaderboards)) {
    self.selected_leaderboard_index = self.competition.leaderboards[0].id
}
```

Reply:
✔️